File: my-folder/3242-count-elements-with-maximum-frequency/solution.py
class Solution(object):
    def maxFrequencyElements(self, nums):
        new_lst=list(set(nums))
        a=[]
        for i in range(len(new_lst)):
            a.append(nums.count(new_lst[i]))
        maxs=max(a)
        c=0
        for i in a:
            if maxs==i:
                c=c+i
        return c    

File: my-folder/3242-count-elements-with-maximum-frequency/solution.java
class Solution {
    public int maxFrequencyElements(int[] nums) {
        HashMap<Integer , Integer> map = new HashMap<>();

        for(int i=0 ; i<nums.length ; i++){
            map.put(nums[i] , map.getOrDefault(nums[i],0)+1);
        }

        int max = Collections.max(map.values());

        int count = 0;
        for(int val : map.values()){
            if(val == max){
                count = count + val;
            }
        }

        return count;
    }
}
